pandas/core/style.py

```diff
@@ -771,3 +850,14 @@ def _highlight_extrema(data, color='yellow', max_=True):
                 extrema = data == data.min().min()
             return pd.DataFrame(np.where(extrema, attr, ''),
                                 index=data.index, columns=data.columns)
+
+
+def _maybe_wrap_formatter(formatter):
+    if not (callable(formatter) or com.is_string_like(formatter)):
+        msg = "Expected a template string or callable, got {} instead".format(
+            formatter)
```

Comment:
if you want to fix, then go ahead and merge (use the ``scripts/merge-pr.py`` tool. note that I will occasionaly edit the generated commit message if it includes too much stuff (e.g. it by default includes everything at the top of the PR). you can do it before it pushes upstream.